### ISSUE TITLE
Send commands to UR5 and Mobile Robot

### ISSUE THREAD

**Issue by jaredescott** [Yuxiang-Ma/2.12-Hololens2#4]

**Comment by young-j-park** [comment]
Here is my thought regarding the list of commands that are likely to be required for our project:

## UR5
- Stop (for emergency)
- Up / Down (for remote-control)
- Left / Right (for pointing?)

## Mobile Robot (for the obstacle avoidance)
- Stop
- Change a driving mode (Manual-pilot / Auto-pilot)

and 

**Approach 1**
- Go straight
- Turn left / right

or 

**Approach 2**
-  Provide waypoints in real-world coordinates

**Comment by young-j-park** [comment]
Although a waypoint method is fancier, I'm not sure it will be easy to select waypoints through camera and convert it to the real-world locations. Instead, we can simply provide proxy waypoint such as go straight / turn left / turn right. I think Approach 1 is much simpler and does not need additional implementation.

If we stick to approach 1, what we need to develop is simply making several buttons on the UI and send the corresponding "pre-defined" message when each one is clicked :)

@jscott19 @Yuxiang-Ma Lmk your thoughts.

**Comment by malekinho8** [comment]
I also agree that Approach 1 is simpler, and we can work together maybe this weekend to try and get this feature implemented.

**Comment by malekinho8** [comment]
(accidentally closed the issue)

**Comment by malekinho8** [comment]
@Yuxiang-Ma I think you were working on this last, but I can go ahead and try to finish it up since I have some idea for implementing it with the server.py script. If you have already made progress on this though, I can work on something else.
